### PR TITLE
Fix translations without named argument

### PR DESCRIPTION
**Pull request by nickpell**
This gets rid of the last translations warnings about strings with multiple unnamed arguments.

### CODE REVIEW

File: custom/ewsghana/alerts/alerts.py [hunk 1]
```diff
@@ -70,6 +70,6 @@ def get_alerts(self, transactions):
 
         if super_message:
             stripped_message = super_message.strip().strip(';')
-            super_message = _('Dear %s, %s is experiencing the following problems: ') + stripped_message
+            super_message = _('Dear %(name)s, %(location)s is experiencing the following problems: ') + stripped_message
```
Comment (dimagimon) [inline custom/ewsghana/alerts/alerts.py]: E501 line too long (120 > 115 characters)

**Comment by nickpell** [comment]
@sravfeyn

File: custom/ewsghana/alerts/__init__.py [hunk 2]
```diff
@@ -3,19 +3,13 @@
...
                          'have not reported for the past month. Please log in to http://ewsghana.com for details.')
 WEB_REMINDER = _('Dear %s, you have not visited ewsghana.com in a long time. '
                  'Please log in to find up-to-date info about stock availability and bottlenecks in Ghana.')
-REPORT_REMINDER = _('Dear %s, %s has not reported its stock this week.'
-                    'Please make sure that the SMS stock report is submitted. ')
-INCOMPLETE_REPORT = _('Dear Name, %s SMS stock report was INCOMPLETE.'
-                      'Please report for: %s')
 COMPLETE_REPORT = _('Dear %s, thank you for reporting the commodities you have in stock.')
-BELOW_REORDER_LEVELS = _('Dear %s, the following commodities are at or below re-order level at the %s. %s')
-ABOVE_THRESHOLD = _('Dear %s, these items are overstocked: %s. The district admin has been informed.')
```
Comment (czue) [inline custom/ewsghana/alerts/__init__.py]: @kkrampa confirming these are no longer needed?

**Comment by czue** [comment]
:+1: . just had one paranoid question then will merge

File: custom/ewsghana/alerts/__init__.py (hunk 2 above)
Comment (kkrampa) [inline custom/ewsghana/alerts/__init__.py]: Double checked and these messages are no longer needed

**Comment by nickpell** [comment]
https://github.com/dimagi/commcare-hq/pull/10417#discussion_r53939597 - looks good to merge

**Pull request merged**